packages/beacon-node/src/chain/archiver/interface.ts

```diff
@@ -0,0 +1,46 @@
+import {CheckpointWithHex} from "@lodestar/fork-choice";
+import {Metrics} from "../../metrics/metrics.js";
+import {RootHex} from "@lodestar/types";
+
+export enum StateArchiveMode {
+  Frequency = "frequency",
+  // Specify only existing strategy
```

Comment:
may want to link the the state diff issue, something like:
// new strategy to be implemented in https://github.com/ChainSafe/lodestar/issue/xxxx